test/e2e/models/CodeServer.ts

```diff
@@ -47,11 +55,18 @@ export class CodeServer {
     return address
   }
 
+  async dir(): Promise<string> {
```

Comment:
nit: we may want to call this `workspaceDir` since I think that accurately describes what this does. I thought it created a new directory at first glance.

Reply:
Agreed!  I might make it not a function as well.